src/main/Config.cpp

```diff
@@ -992,6 +992,10 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
                 throw std::invalid_argument(err);
             }
         }
+
+        // Validators default to starting the network from local state
+        FORCE_SCP = NODE_IS_VALIDATOR;
```

Comment:
grep for `force-scp`; it looks like you need a few updates:
* comments in config.h
* commands.md - (need to document instead the new option command)
* testnet.md - behavior described is wrong, also needs to be updated to describe when to use the new option

Reply:
Good call, updated. Next thing I'm going to do is try it with several nodes like you suggested, and also see if this change breaks anything in the testing pipeline.